assets/app/scripts/extensions/javaLink.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular
-  .module('openshiftConsoleExtensions')
+  .module('openshiftConsoleExtensions', ['openshiftConsole'])
```

Comment:
Could also just keep it a self-contained module, extensions could either use their own angular module or use a shared one.  At the time I just picked that name for lack of a better one :-) but could just call it 'javaLinkExtension' or something too, just need to ensure that same module name is passed to `hawtioPluginLoader.addModule()`.